### ISSUE TITLE
Make examples available independent of the code

### ISSUE THREAD

**Issue by dalonsoa** [SGIModel/MUSE_OS#162]
[In the tutorials section of the documentation](https://muse-os.readthedocs.io/en/flow_charts/user-guide/add-solar.html), it is regularly indicated that users should navigate to the the location of the examples WITHIN THE SOURCE CODE. Eg:

```md
In this section, we will add solar photovoltaics to the default model seen in the example page. To achieve this, we must modify some of the input files shown in the above section. These files can be found in the StarMuse folder at the following location:

`{muse_install_location}/src/muse/data/example/default`
```
(note that `StarMuse` should be replaced by MUSE, too)

 **This is plainly wrong**: end users should not need to look for anything within MUSE source code - they do not even need to know where such a source code is. Moreover, if the GUI version of MUSE (the current one of a future one) becomes popular, there will be no source code to look into as the end user will only get binary files. 

The cleanest solution will be to separate [the examples folder](https://github.com/SGIModel/MUSE_OS/tree/develop/src/muse/data) to its own repository, pointing users to it, and make it a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules), so it is still usable in the normal way from MUSE. This will make life slightly more complicated for developers, but hopefully it will not be too much of a problem on a regular basis as the examples submodule is not likely to be updated regularly. 

The same applies to the[ Tutorial Code](https://github.com/SGIModel/MUSE_OS/tree/develop/docs/tutorial-code) for the tutorials included in the documentation. There is no way for the users to just download that tutorial code and start experimenting: they need to download the whole of MUSE source code.

@sgiarols @ahawkes @martinstringer , it will be good to have your comments on this before making any changes, which will need to be accompanied by appropriate changes in the documentation, obviously.

**Comment by sgiarols** [comment]
It is a nice idea to separate code from examples. 
One possible alternative could be to have the data on zenodo rather than another git repository.

**Comment by sgiarols** [comment]
apologies, I did not mean to close the issue

**Comment by dalonsoa** [comment]
It's my fault. I gave the wrong issue number in the issue mentioned above so it was closed automatically when the PR was merged.

**Comment by dalonsoa** [comment]
No longer required to fix the issue mentioned above by means of #180
